shared/src/test/factories.ts

```diff
@@ -457,6 +457,7 @@ export const generateBusiness = (overrides: Partial<Business>): Business => {
     taxFilingData: generateTaxFilingData({}),
     environmentData: generateEnvironmentData({}),
     version: CURRENT_VERSION,
+    userId: generateUser({}).id,
```

Comment:
This isn't going to break functionality today, but in the future we're going to start relying on the `userId` in the business matching the user's ID. We may want `generateBusiness` to optionally take a user so that it can extract the ID here.

Reply:
agreed that this will be addressed in a later ticket!